contracts/test/AllowanceTarget/SetupTest.t.sol

```diff
@@ -1,15 +1,19 @@
-// SPDX-License-Identifier: UNLICENSED
+// SPDX-License-Identifier: MIT
 pragma solidity 0.7.6;
 
 import "contracts-test/AllowanceTarget/Setup.t.sol";
 
-contract TestAllowanceTargetConstructor is TestAllowanceTarget {
+contract TestAllowanceTargetSetup is TestAllowanceTarget {
+    function testAllowanceTargetSetup() public {
+        assertEq(allowanceTarget.spender(), address(this));
+    }
```

Comment:
This is also the test of constructor? Maybe the `Test setup: constructor` comment  should be put before this test?

Reply:
Yes, it is also similar to test which creates a new contract in Setup.t.sol...
Or should we remove the `Test setup: constructor` comment to reduce code?

Reply:
Yeah I think now that tests are separated into files, the comment can be removed.

Reply:
OK! Thank you!